pkg/controller/manager/types_old.go

```diff
@@ -37,16 +38,18 @@ func (r RepairTask) ToManager() (*managerclient.Task, error) {
 		return nil, errors.Wrap(err, "parse start date")
 	}
 	t.Schedule.StartDate = &startDate
-
-	if _, err := duration.ParseDuration(r.Interval); err != nil {
-		return nil, errors.Wrap(err, "parse interval")
-	}
```

Comment:
is it part of webhook validation? If not, please move it there.

Reply:
I'll revert this change here as it's unrelated and I'll open a separate PR for it. Preferably after https://github.com/scylladb/scylla-operator/pull/1853 and https://github.com/scylladb/scylla-operator/pull/1850 land.

Reply:
reverted